setup.py

```diff
@@ -61,16 +60,9 @@
     'numpy>=1.9.2',
     'pandas>=0.16.1',
     'scipy>=0.15.1',
-    "pandas-datareader>=0.2",
-    'bottleneck>=1.0.0'
```

Comment:
If we're removing it here we should either list is an optional or add to the readme that people could install bottleneck in addition for more speed.